src/bite.v1.js

```diff
@@ -21,9 +22,41 @@
         request.send()
     }
 
+    function determineSession() {
+        const cookies = document.cookie
+            .split('; ')
+            .map(cookie => cookie.split('='))
+            .reduce((mapping, [key, value]) => ({
+                ...mapping,
+                [key]: value,
+            }), {})
+
+        if (cookies['barnebys_session']) {
+            return cookies['barnebys_session']
+        }
+
+        const currentLocation = new URL(window.location)
+        const searchParams = new URLSearchParams(currentLocation.search)
+        const params = Array.from(searchParams.entries())
+        const btmTags = params
```

Comment:
This includes all the parameters starting with `btm_`, which means that we could handle `btm_locale` and other things as well, just not sure what to do with it just yet